tests/e2e/containers/containers.go

```diff
@@ -212,8 +212,10 @@ func (m *Manager) RunNodeResource(chainId string, containerName, valCondifDir st
 // The genesis and configs are to be mounted on the init container as volume on mountDir path.
 // Returns the container resource and error if any. This method does not Purge the container. The caller
 // must deal with removing the resource.
-func (m *Manager) RunChainInitResource(chainId string, chainVotingPeriod int, validatorConfigBytes []byte, mountDir string, forkHeight int) (*dockertest.Resource, error) {
+func (m *Manager) RunChainInitResource(chainId string, chainVotingPeriod, chainExpeditedVotingPeriod int, validatorConfigBytes []byte, mountDir string, forkHeight int) (*dockertest.Resource, error) {
 	votingPeriodDuration := time.Duration(chainVotingPeriod * 1000000000)
+	// TODO: Uncomment this after v12 release
```

Comment:
Let's make an issue for this, please

Reply:
https://github.com/osmosis-labs/osmosis/issues/2386, I can likely take this post v12